### ISSUE TITLE
3.5.0 Unknown browser query `dead`

### ISSUE THREAD

**Issue by Nomia** [sampotts/plyr#1350]
Error in version 3.5.0 when building my project, version 3.4.7 is ok.

```
ERROR in ./node_modules/plyr@3.5.0@plyr/dist/plyr.css
Module build failed: ModuleBuildError: Module build failed: BrowserslistError: Unknown browser query 'dead'
```

**Comment by sampotts** [comment]
The query is `not dead` and it builds fine here. 
https://github.com/sampotts/plyr/blob/master/package.json#L32

Perhaps you need to update something on your end? It's valid:
https://browserl.ist/?q=%3E+1%25%2C+not+dead

**Comment by Nomia** [comment]
Thx,
I don't know the exact reason for this.
(I used some npm mirror for my `npm install` process, the registry is `https://registry.npm.taobao.org`)

My temporary fix is change the version in my `package.json` file:
from
```
"dependencies": {
    "plyr": "^3.4.7"
  },
```
to
```
"dependencies": {
    "plyr": "~3.4.7"
  },
```
version below 3.5.0 is ok.

**Comment by sampotts** [comment]
This should be fixed in v3.5.1. Cheers 👍
